module-1/web-project/your-code/FILMAFFINITY_WEB_SCRAPING.ipynb

```diff
@@ -1,55 +1,92 @@
 {
```

Comment:
Amazing job here Ivan!! Well done!! As in API section once you have proved your code works in a procedural way try to convert it to a function based code (when possible).